views/includes/corporate-directors.html

```diff
@@ -6,23 +7,23 @@ <h2 class="govuk-summary-card__title">{{officerCard.officer.name}}</h2>
     <dl class="govuk-summary-list">
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          Role
+          {{ i18n.currentDirectorRoleLabel }}
         </dt>
         <dd class="govuk-summary-list__value">
-          Director
+          Director        
         </dd>
       </div>
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          Date appointed
+          {{ i18n.currentDirectorDobLabel }}
```

Comment:
I think this should be currentDirectorAppointedDateLabel rather than the dob one

Reply:
Good spot. fixed